content/edpratti/collections/git-guide/posts/git-into-the-weeds/index.md

```diff
@@ -233,8 +242,13 @@ git commit -m "Updating MaterialUI"
 
 Submodules can create very complex repository structures, so it's important to understand when they are needed. For most situations, developers have moved on to package managers, which are friendlier to manage than submodules. In fact, the example we've shown today — MaterialUI - is actually available as a package!
 
-We have a great article on setting up NPM and Yarn. [**You can read it by clicking here**](/posts/how-to-use-npm). 
+> **Introduction to NPM:**
+> We have a great article on setting up NPM and Yarn. [**You can read it by clicking here**](/posts/how-to-use-npm). 
 
 ---
 
-<!-- TO-DO: Add conclusion -->
\ No newline at end of file
+# What to expect next
+
+In our next chapter, or perhaps chapters, we'll approach more advanced topics, like Git `reflog`, `gc`, `fsck` and the `.git` folder.
+
+#### We'll see you then!
```

Comment:
Ending on a heading like this is a bad idea for A11Y, let's make it a bolded text instead